packages/react-heartwood-components/src/components/Core/components/HeaderPrimary/HeaderPrimary.js

```diff
@@ -23,7 +23,13 @@ type Props = {
 	toggleSidebarVisibility: Function,
 
 	/** Set true to show the sidebar (small screens only) */
-	sidebarIsVisible: boolean
+	sidebarIsVisible: boolean,
```

Comment:
@gcpantazis consider renaming to `isSidebarVisible` to follow convention we discussed.